### PR TITLE
Update publications

### PR DESCRIPTION
**Pull request by lukasmittag**

### CODE REVIEW

File: content/publications.md [hunk 1]
```diff
@@ -4,31 +4,34 @@ date: 2022-08-12T09:46:31+02:00
...
+- 12/2019 [APPSTACLE and Eclipse KUKSA Interview](https://www.youtube.com/watch?v=-dOwFY2l-SY) with Sven Erik Jeroschewski (Bosch Software Innovations) and Philip Heisig (Dortmund University of Applied Sciences and Arts) at EclipseCon Europe 2019
+- 12/2019 [Using Eclipse Kuksa for Connected Vehicle Services](https://www.youtube.com/watch?v=FcqzLn-wiB0) by Sven Erik Jeroschewski, Pratheek Rai adn Sebastian Lohmeier at EclipseCon Europe 2019
+- 05/2019 [Bridging Eclipse SUMO with Eclipse Kuksa](https://youtu.be/FuIaJ2tlnRE) by Philipp Heisig, Fachhochschule Dortmund
+- 03/2019 [Appstacle Use Case: Remote Driver Authentication](https://youtu.be/eZHSCXSc2wc). KUKSA PoC by OTOKAR and Netaş
+- 11/2018 [Automotive meets IoT: Innovating your future vehicle](https://www.youtube.com/watch?v=tD8pt7WMbuQ) by Sebastian Schildt, Bosch
+- 05/2018 [APPSTACLE interview](https://www.youtube.com/watch?v=qCyBjZnGK2E) with Robert Höttger Fachhochschule Dortmund - University of Applied Science and Art at Eclipsecon France
+- 03/2023 [Vehicle Signaling Specification and KUKSA.val in AGL](https://www.youtube.com/watch?v=RhSocQDu_DY) by Scott Murray, Konsulko Group at AGL All Member Meeting
```
Comment (SebastianSchildt) [inline content/publications.md:18]: You added the newest stuff to the wrong side (It is at the bottom)

File: content/publications.md [hunk 2]
```diff
@@ -4,31 +4,34 @@ date: 2022-08-12T09:46:31+02:00
...
-* M. Wagner, S.Schildt, Innovation durch Offenheit: Das Open Source Connected Vehicle Framework Eclipse Kuksa, Boardnetz Kongress 2019, https://www.researchgate.net/profile/Marco-Wagner/publication/330281127_Innovation_through_Openness_-_The_Open_Source_Connected_Vehicle_Framework_Eclipse_Kuksa/links/5c371bcda6fdccd6b5a0a357/Innovation-through-Openness-The-Open-Source-Connected-Vehicle-Framework-Eclipse-Kuksa.pdf
-* Banijamali A., Jamshidi P., Kuvaja P., Oivo M.:
-"Kuksa: A Cloud-Native Architecture for Enabling Continuous Delivery in the Automotive Domain",
-Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics), 2019,
-https://link.springer.com/chapter/10.1007/978-3-030-35333-9_32
-* Höttger R., Özcelikörs M., Heisig P., Krawczyk L., Cuadra P., Wolff C.: "Combining eclipse IoT technologies for a RPI3-Rover along with Eclipse Kuksa",
-CEUR Workshop Proceedings 2018, http://ceur-ws.org/Vol-2066/seerts2018paper03.pdf
\ No newline at end of file
+
```
Comment (SebastianSchildt) [inline content/publications.md:22]: Any changes here? Just curious github sees a diff :)

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
See comments

File: content/publications.md (hunk 1 above)
Comment (lukasmittag) [reply content/publications.md:18]: Oh yes, didn't look closely enough :)

File: content/publications.md (hunk 2 above)
Comment (lukasmittag) [reply content/publications.md:22]: code formatter added this line :)

**Review by SebastianSchildt — APPROVED** [review]

**Pull request merged**